src/frontend/src/features/rooms/utils/generateRoomId.ts

```diff
@@ -1,5 +1,30 @@
-import { slugify } from '@/utils/slugify'
+
+const getRandomChar = () => {
+  // Google Meet uses only letters in a room identifier
```

Comment:
so we officialy are google-meet driven development huh? :smirk:

Reply:
We totally remove this mention if you feel it's weird